great-idea-website/css/index.css

```diff
@@ -63,4 +63,73 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
```

Comment:
You might want to add a container or stying to the body with some max-width to make sure your style looks good on bigger screen as well